analyses/sample-distribution-analysis/01-filter-across-types.R

```diff
@@ -0,0 +1,225 @@
+# This script filters the given dataset to produce a summarized visualization
+# of key variables within the dataset.
+#
+# Chante Bethell for CCDL 2019
+
+
+# Load/install packages. This will be removed once a Dockerfile is established.
+source("00-install-packages.R")
```

Comment:
I am looking forward to having this removed because if you are installing a new package, it is also being loaded into the namespace which may or may not be what you want to do.